dev/release/utils-prepare.sh

```diff
@@ -194,6 +194,7 @@ update_versions() {
                      "${base_version}" \
                      "${next_version}"
   git add docs/source/_static/versions.json
+  git add r/pkgdown/assets/versions.html
```

Comment:
Aaah was this missing so we weren't actually updating that page? Oops!

Reply:
Right. :-)